include/z64item.h

```diff
@@ -371,4 +372,12 @@ typedef enum {
     /* 0x68 */ GID_SWORD_GREAT_FAIRY
 } GetItemDrawID;
 
+// TODO: fill
+typedef enum {
+    /*   -1 */ EXCH_ITEM_MINUS1 = -1, // Unknown usage or function
+    /* 0x00 */ EXCH_ITEM_NONE,
+    /* 0x1E */ EXCH_ITEM_1E = 0x1E, // BOTTLE_MUSHROOM
+    /* 0x2A */ EXCH_ITEM_2A = 0x2A // MOON_TEAR?
```

Comment:
Because I don't completely understand the exchange item system, and last time we discussed the `exchangeItemId` member of `Player` I ended up more confused that how we started. Hopefully decomping player would help there